DEVELOPMENT.md

```diff
@@ -336,6 +336,23 @@ changes to existing stuff are required for a new feature.
 Prior to pgai v0.4.0, Python dependencies were installed system-wide. Until pgai versions 0.1 - 0.3 are deprecated
 [old dependencies](./src/old_requirements.txt) are installed system-wide.
 
+#### Release the extension
+
+1. Run `just ext clean-sql` in order to remove any prior generated dev sql.
+2. Edit the [build.py](projects/extension/build.py).
+   - Set the version to be released as the first element of the array returned by `versions` function. I.e. `"0.7.0",  # released`.
+3. Edit the extension [__init__.py](projects/extension/ai/__init__.py) and set the value of the `__version__` variable to your new version . I.e. `__version__ = "0.7.0"`.
+4. Run `just ext build`. This will generate, among other tasks, the final .sql files in `projects/extension/sql`.
```

Comment:
```suggestion
3. Run `just ext build install`. This will generate, among other tasks, the final .sql files in `projects/extension/sql` and update the version in [__init__.py](projects/extension/ai/__init__.py).
```

Reply:
updating the version in `__init__.py` is handled automatically by [build_init_py](https://github.com/timescale/pgai/blob/main/projects/extension/build.py#L509) but we have to run the install step for that to execute

Reply:
let's make a new step that just does [build_sql](https://github.com/timescale/pgai/blob/main/projects/extension/build.py#L338) and [build_init_py](https://github.com/timescale/pgai/blob/main/projects/extension/build.py#L509) then.